scaling/iam.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_iam_role" "ecs-autoscale-role" {
+  name = "ecs-scale-${terraform.env}-${var.cluster_name}"
```

Comment:
If this module is aimed at configuring the service auto scaling, do not create cluster specific resources in this module. Trying to call this module to configure the service scaling for two services on the same cluster will lead to an error when calling it the second time as the name is already taken. The cluster resources should be created when in the module setting up the ECS cluster.